res/skins/QMLDemo/VuMeter.qml

```diff
@@ -0,0 +1,82 @@
+import Mixxx 0.1 as Mixxx
+import QtGraphicalEffects 1.12
+import QtQuick 2.12
+import "Theme"
+
+Rectangle {
+    id: root
+
+    property string group // required
+    property string key // required
+    property color barColor // required
+
+    radius: width / 2
```

Comment:
for performance reasons, we could consider disabling antialiasing for this Rectangle because i don't think it makes a difference here. https://doc.qt.io/qt-5/qml-qtquick-rectangle.html#performance